doc/man3/SSL_set_initial_peer_addr.pod

```diff
@@ -0,0 +1,53 @@
+=pod
+
+=head1 NAME
+
+SSL_set_initial_peer_addr - set the initial peer address for a QUIC connection
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ __owur int SSL_set_initial_peer_addr(SSL *s, const BIO_ADDR *addr);
+
+=head1 DESCRIPTION
+
+SSL_set_initial_peer_addr() sets the initial destination peer address to be used
+for the purposes of establishing a QUIC connection in client mode. This function
+can be used only on a QUIC connection SSL object, and can be used only before a
+connection attempt is first made. B<addr> must point to a B<BIO_ADDR>
+representing a UDP destination address of the server to connect to.
```

Comment:
Hmm. AIUI, an application can DTLS a `BIO_s_dgram` which uses a FD which has had `connect()` called on it (and `BIO_s_dgram` uses send/recv). Or, it can give DTLS a `BIO_s_dgram` which has had the peer set on it (and `BIO_s_dgram` uses sendto/recvfrom). So that's two cases to handle. What would you like `SSL_set_initial_peer_addr` to do when called on a DTLS SSL object? As far as I can tell very little of our DTLS code is really aware of L4 addresses. It just works with a blind point-to-point datagram stream.